lambda-layer/packages/layer/scripts/otel-instrument-esm

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+export NODE_OPTIONS="${NODE_OPTIONS} --import @aws/aws-distro-opentelemetry-node-autoinstrumentation/register --experimental-loader=@opentelemetry/instrumentation/hook.mjs"
+export LAMBDA_RESOURCE_ATTRIBUTES="cloud.region=$AWS_REGION,cloud.provider=aws,faas.name=$AWS_LAMBDA_FUNCTION_NAME,faas.version=$AWS_LAMBDA_FUNCTION_VERSION,faas.instance=$AWS_LAMBDA_LOG_STREAM_NAME,aws.log.group.names=$AWS_LAMBDA_LOG_GROUP_NAME";
+export IS_ESM=true
+
+# - If OTEL_EXPORTER_OTLP_PROTOCOL is not set by user, the default exporting protocol is http/protobuf.
+if [ -z "${OTEL_EXPORTER_OTLP_PROTOCOL}" ]; then
+    export OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+fi
+
+# - If OTEL_NODE_ENABLED_INSTRUMENTATIONS is not set by user, use default instrumentation
+if [ -z "${OTEL_NODE_ENABLED_INSTRUMENTATIONS}" ]; then
+    export OTEL_NODE_ENABLED_INSTRUMENTATIONS="aws-lambda,aws-sdk"
+fi
+
+# - Set the service name
+if [ -z "${OTEL_SERVICE_NAME}" ]; then
+    export OTEL_SERVICE_NAME=$AWS_LAMBDA_FUNCTION_NAME;
+fi
+
+# - Set the propagators
+if [[ -z "$OTEL_PROPAGATORS" ]]; then
+  export OTEL_PROPAGATORS="tracecontext,baggage,xray"
```

Comment:
In `register.ts`, we use `xray,tracecontext,b3,b3multi`. Curious why there's a difference here?

Reply:
We're just following the existing OTel-lambda layer behavior on propagator settings for now
https://github.com/open-telemetry/opentelemetry-lambda/blob/51c8d3026742d1095f7f7614e88856c02dde49c4/nodejs/packages/layer/scripts/otel-handler#L12